trunk-recorder/main.cc

```diff
@@ -1756,15 +1758,6 @@ int main(int argc, char **argv) {
 
   start_plugins(sources, systems);
 
-  if (config.log_file) {
-    logging::add_file_log(
-        keywords::file_name = config.log_dir + "/%m-%d-%Y_%H%M_%2N.log",
-        keywords::format = "[%TimeStamp%] (%Severity%)   %Message%",
-        keywords::rotation_size = 100 * 1024 * 1024,
```

Comment:
Do these lines need to be brought up? Looks like we are losing them

Reply:
these lines were duplicated at lines 161 and 162 in #754 and should have been deleted here

Reply:
Gotcha!